homeassistant/components/websocket_api/permissions.py

```diff
@@ -38,7 +35,6 @@
     EVENT_ENTITY_REGISTRY_UPDATED,
     EVENT_LOVELACE_UPDATED,
     EVENT_PANELS_UPDATED,
-    EVENT_PERSISTENT_NOTIFICATIONS_UPDATED,
```

Comment:
I don't know which things subscribe to this outside HA, but maybe we should keep it in for 2-3 releases to avoid it raising Unauthorized

Reply:
persistent_notifications_no_more_state_machine